CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-## 2.16.1 (Unreleased)
+## 2.16.2 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.1** (tested on **UserConnect-6.1.1276**)
```

Comment:
Supported controller version should be specific since this is a patch. Unsure what the latest version is since 6.1.patch not released yet